prestodb/presto
# [native] Make memory reclaim timeout configurable

## Description

Allow changing memory reclaim timeout

## Motivation and Context

Sometimes it is useful to increase it or set it to 0 to effectively disable it and have the process wait forever for further diagnosis

## Impact

N/A

## Test Plan

- Run TestPrestoSparkNativeAggregations
- Check the configuration property is added to config.properties and applied by the native process

## Contributor checklist


## Release Notes
Please follow [release notes guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines) and fill in the release notes below.


```
== NO RELEASE NOTE ==
```
